app/src/main/res/values/colors.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <color name="blue">#4a90e2</color>
+    <color name="red">#f2655d</color>
+    <color name="white">#ffffff</color>
+    <color name="green">#63c774</color>
```

Comment:
this should be alphabetized, and you should either stick with `colorPrimary` naming conventions, or `blue`, but not mix and match both